tests/ttnn/unit_tests/test_reshape.py

```diff
@@ -285,3 +285,16 @@ def test_reshape_tile_layout_only_change_shape(device):
     output = ttnn.to_torch(ttnn_output)
 
     assert_with_pcc(torch_result, output, 0.9999)
+
+
+def test_reshape_bert(device):
```

Comment:
can we call it smth different? not just bert, but whats the semantics. 
test_reshape_rowmajor_change_row_size